src/cmd_line/commandLine.ts

```diff
@@ -38,14 +38,17 @@ export class CommandLine {
         await cmd.execute(vimState.editor, vimState);
       }
     } catch (e) {
-      console.log(e);
       if (e instanceof VimError) {
-        StatusBar.SetText(
-          `${e.toString()}. ${command}`,
-          vimState.currentMode,
-          vimState.isRecordingMacro,
-          true
-        );
+        if (e.code === ErrorCode.E492) {
+          await Neovim.command(vimState, command);
```

Comment:
Even for users that haven't configured to use neovim, you want to use neovim as a fallback

Reply:
My bad. You're right, there should be another check here.